src/picomsg/src/loggy_system_events/loggy_system_events.cpp

```diff
@@ -0,0 +1,95 @@
+
```

Comment:
Why have a "loggy" backend? Are you really going to use it?
Unlike oper DB, system events are very easy to mock (see routes_conn). I don't see a reason to have a "loggy" option.

Reply:
I used it to see that the strongswan_tests env runs correctly, but I understand why it is not necessary.
I've removed it, thanks